apply/default_applier.go

```diff
@@ -39,6 +43,7 @@ const (
 	ApplyNodes     ActionName = "ApplyNodes"
 	Guest          ActionName = "Guest"
 	Reset          ActionName = "Reset"
+	UnMountCluster ActionName = "UnMountCluster"
```

Comment:
UnMountCluster and Unmount are not easy to distinguish. We really need this function?